spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -153,11 +153,11 @@ Similarly, the sum of `ft.<name>.circuitbreaker.timeOpen.total`, `ft.<name>.circ
 | Histogram | Nanoseconds
 | Histogram of method execution times. This does not include any time spent waiting in the bulkhead queue.
 
-|`ft.<name>.bulkhead.queuePopulation`*
+|`ft.<name>.bulkhead.waitingQueue.population`*
 | Gauge<Long> | None
 | Number of executions currently waiting in the queue
 
-|`ft.<name>.bulkhead.waitDuration`*
+|`ft.<name>.bulkhead.waitingQueue.duration`*
```

Comment:
`ft.<name>.bulkhead.waitingQueue.duration` => `ft.<name>.bulkhead.waiting.duration` as it describes the execution and is not measuring the waiting queue

Reply:
Done.